### DISCUSSION TITLE
How does Dogecoin handle 'transaction' and 'account' in UTXO model

### DISCUSSION THREAD

**Discussion by shiatcb** [rosetta-dogecoin/rosetta-dogecoin#64] [Q&A]
Hi Team,

I am reading the codebase but would love to see if someone could shed some light on how does Dogecoin handle transaction and account in its Rosetta implementation. Specifically:
- Does Dogecoin expect each transaction to be associated with an account (my guess is no for UTXO chain but not very sure)?
- Does Dogecoin store the transaction as it is in each block?
- I saw Dogecoin implements /account/xxx APIs as other chains, e.g: /account/balance. But I don't quite understand why a UTXO chain like Dogecoin needs account info in its rosetta implementation. Is this just because Rosetta spec requires us to do so, or some other reasons?

Some questions might be silly as I am not an expert on UTXO chain. If someone could explain a bit about my questions above, it'll be super appreciated!

Thanks,
Shi

**Comment by patricklodder** [comment]
Hi! I'm traveling and slow but will answer this; it's on my to-do. Apologies.

**Reply by shiatcb** [reply]
Cool, thanks!
